portal/src/graphql/portal/LoginIDConfigurationScreen.tsx

```diff
@@ -547,6 +648,19 @@ const AuthenticationLoginIDSettingsContent: React.FC<AuthenticationLoginIDSettin
         <Label className={styles.checkboxLabel}>
           <FormattedMessage id="LoginIDConfigurationScreen.email.domainAllowlist" />
         </Label>
+        <TagPicker
+          inputProps={{
+            "aria-label": renderToString(
+              "LoginIDConfigurationScreen.email.domainAllowlist"
+            ),
+            placeholder: "example.com",
+          }}
+          className={styles.widgetInputField}
+          disabled={!state.email.domain_allowlist_enabled}
+          selectedItems={domainAllowlist}
+          onChange={onDomainAllowlistChange}
+          onResolveSuggestions={onDomainAllowlistSuggestions}
+        />
```

Comment:
It would be great if we can warn the developer that allowlist is enabled but the allowlist is empty. If the allowlist is enabled and is empty, signup is not possible. Maybe we can display the warning as error under the control?

Reply:
Added tooltip to labels